### PR TITLE
feat: admin crud

### PR DESCRIPTION
**Pull request by ZulfanoRp**

### CODE REVIEW

File: app/Http/Controllers/admin/AdminController.php [hunk 1]
```diff
@@ -23,8 +23,9 @@ public function listData()
 
     public function postUser(Request $req)
     {
+        $uuid = DB::select("SELECT uuid() as uuid");
```
Comment (KrishnaHendra) [inline app/Http/Controllers/admin/AdminController.php:26]: iki dihilangkan, engko nggawe function general seng fungsi e nge return uuid dewe cek reusable

File: resources/views/admin/pages/list-data.blade.php [hunk 2]
```diff
@@ -152,13 +151,22 @@ class="col-lg-4 col-form-label"
                                                                                                 class="text-danger">*</span>
                                                                                         </label>
                                                                                         <div class="col-lg-8">
-                                                                                            <input type="text"
+                                                                                            <input type="password"
                                                                                                 class="form-control"
-                                                                                                id="val-username"
-                                                                                                name="val-username"
+                                                                                                id="password-confirmation"
```
Comment (KrishnaHendra) [inline resources/views/admin/pages/list-data.blade.php:156]: id & name e diganti nggawe undescore, ex: password_confirmation

File: app/Http/Controllers/admin/AdminController.php (hunk 1 above)
Comment (ZulfanoRp) [reply app/Http/Controllers/admin/AdminController.php:26]: siap oke

File: resources/views/admin/pages/list-data.blade.php (hunk 2 above)
Comment (ZulfanoRp) [reply resources/views/admin/pages/list-data.blade.php:156]: nice info

File: app/Http/Controllers/admin/AdminController.php (hunk 1 above)
Comment (KrishnaHendra) [reply app/Http/Controllers/admin/AdminController.php:26]: public function getUUID(){
    // get uuid from db
    return uuid;
}

File: app/Http/Controllers/admin/AdminController.php [hunk 3]
```diff
@@ -16,31 +17,8 @@ public function index()
...
-        ]);
-    }
-    
-    public function getUUID(){
-        $data = DB::select("SELECT uuid() as uuid_value");
-        $uuid = $data[0]->uuid_value;
-        return $uuid;
+        // $data["plan"] = Plan::all();
+        $data["listData"] = User::all();
+        return view("admin.pages.list-data", $data);
```
Comment (KrishnaHendra) [inline app/Http/Controllers/admin/AdminController.php:22]: mantap brodi langsung merge habis ini

Comment (ZulfanoRp) [reply app/Http/Controllers/admin/AdminController.php:22]: okay, later before lunch, i will finished delete features

**Pull request merged**